.github/workflows/frontend.yml

```diff
@@ -186,7 +177,7 @@ jobs:
       # it reduces large coverage fluctuations.
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
-        if: ${{ always() && needs.files-changed.outputs.frontend_all == 'true' }}
```

Comment:
The comment above suggests we should do this all the time, and the `frontend_src == 'true'` check is moved up in the tree, not needed here anymore.